PlaygroundService/Startup.cs

```diff
@@ -1,12 +1,37 @@
 using Microsoft.AspNetCore.Builder;
 using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.Configuration;
 using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
-
+using MongoDB.Driver;
+using MongoDB.Driver.GridFS;
+using System;
 public class Startup
 {
-    public void ConfigureServices(IServiceCollection services) =>
+    public IConfiguration Configuration { get; }
+
+    public Startup(IConfiguration configuration)
+    {
+        Configuration = configuration;
+    }
+
+    public void ConfigureServices(IServiceCollection services)
+    {
+        // Check if Configuration is properly loaded
+        if (Configuration == null)
+        {
+            throw new Exception("Configuration is null. Ensure it is initialized correctly.");
+        }
+
+        var mongoClient = new MongoClient(""); // DB String
+        var mongoDatabase = mongoClient.GetDatabase(""); // TabelName
+        var gridFSBucket = new GridFSBucket(mongoDatabase);
```

Comment:
I think you might need to use Orlean's MongoDB Clustering instead to be compatible with Orleans Grain for scalability.

[Orleans MongoDB](https://github.com/OrleansContrib/Orleans.Providers.MongoDB)